code/tasks/FTFileMakerTask.php

```diff
@@ -62,31 +63,69 @@
  */
 class FTFileMakerTask extends BuildTask
 {
+    /**
+     * If set to TRUE skip thumbnail generation
+     * @var bool
+     * @config
+     */
+    private static $documentsOnly = false;
 
-    protected $documentsOnly = false;
-
-    protected $doSetFolderPermissions = false;
+    /**
+     * Vary the permission on the folders
+     * @var bool
+     * @config
+     */
+    private static $doSetFolderPermissions = true;
 
-    // simulate scenario where protected files were uploaded into the wrong asset store
-    protected $doPutProtectedFilesInPublicStore = false;
+    /**
+     * Put some files in wrong store to test logic meant to correct this kind of problem..
+     * @var bool
+     * @config
+     */
+    private static $doPutProtectedFilesInPublicStore = false;
 
-    protected $doSetOldCreationDate = true;
+    /**
+     * Set the date of some files to an old date.
+     * @var bool
+     * @config
+     */
+    private static $doSetOldCreationDate = true;
 
-    protected $doRandomlyPublish = true;
+    /**
+     * Publish some files.
+     * @var bool
+     * @config
+     */
+    private static $doRandomlyPublish = true;
 
-    protected $depth = 2;
+    /**
+     * How deep should or folder hierachy be.
+     * @var int
+     * @config
+     */
+    private static $depth = 2;
 
-    protected $folderCountByDepth = [
+    /**
+     * Number of folders to create certain hierachy.
+     * @var int[]
+     * @config
+     */
+    private static $folderCountByDepth = [
         0 => 2,
         1 => 2,
         2 => 2,
         3 => 2,
         4 => 2,
     ];
 
-    protected $fileCountByDepth = [
-        0 => 10,
-        1 => 8,
+    /**
+     * Number of files to create at various depths in the hierachy
+     * @var int[]
+     * @config
+     */
+    private static $fileCountByDepth = [
+        0 => 100,
```

Comment:
This restore the old values here. I think we need a decent amount of files by default even if it takes a faire bit of tiem to run.

Reply:
We also need a way to quickly test the pagination.